code/__defines/items_clothing.dm

```diff
@@ -101,11 +101,9 @@
 #define slot_r_ear_str      "slot_r_ear"
 #define slot_belt_str       "slot_belt"
 #define slot_shoes_str      "slot_shoes"
-#define slot_head_str      	"slot_head"
```

Comment:
i added these, don't remove them

Reply:
Higher up in this dm is this same defined repeated.
![image](https://user-images.githubusercontent.com/49106866/120087652-be593e80-c0b7-11eb-95a8-b79dfb2b6dcd.png)